src/components/InputEditor.tsx

```diff
@@ -32,11 +32,8 @@ function getIssueReportUrl({
   const reportUrl = new URL(
     `https://github.com/swc-project/swc/issues/new?assignees=&labels=C-bug&template=bug_report.yml`
   )
-  if (code.length > 2000)
-    code =
-      'Your input is too large to share. Please copy the code and paste it in the issue.'
 
-  reportUrl.searchParams.set('code', code)
+  reportUrl.searchParams.set('code', code.length > 2000 ? '' : code)
```

Comment:
Why do you clear the code? Whether the code is too long or not, it shouldn't be modified.